src/server/services/training.service.ts

```diff
@@ -121,14 +128,39 @@ export const createTrainingRequest = async ({
     }
   }
 
+  const eta = calcEta(
+    trainingParams.networkDim,
+    trainingParams.networkAlpha,
+    trainingParams.targetSteps,
+    baseModel
+  );
+  const price = eta !== undefined ? calcBuzzFromEta(eta) : eta;
+  if (price === undefined) {
+    throw throwBadRequestError(
+      'Could not compute Buzz price for training - please check your parameters.'
+    );
+  }
```

Comment:
```suggestion
  if (!eta) {
    throw throwBadRequestError(
      'Could not compute Buzz price for training - please check your parameters.'
    );
  }
  const price = calcBuzzFromEta(eta);
```

Reply:
i mean this is kind of the same thing, but this way won't catch an undefined price in the event we change that function.